### PR TITLE
Add RiskScore Component to the Bits package #639

### PR DESCRIPTION
**Pull request by MykhailoMolyboha**
## Related Jira item

https://swicloud.atlassian.net/browse/OO-23867

## Issue
https://github.com/solarwinds/nova/issues/639

## Description
Add Fusion shared 'Risk Score' component

## Screenshots (if applicable)
mockup
image

/components/RiskScoreComponent.html#example
image

## Definition of done

- [x] Unit and E2E tests are written (or will be in scope of another PR)
- [x] Feature toggling is applied - Not Applicable
- [ ] Performance impact was considered
- [x] The security and privacy aspect has been considered - Not Applicable
- [x] Demo mode is considered - Not Applicable

### CODE REVIEW

**Comment by Blakesalot** [comment]
@MykhailoMolyboha, I'm no longer a Nova maintainer so I have no official say in this, but in my opinion this component is very feature-specific and should live somewhere else besides the Nova libraries. The set of components in Nova has been carefully curated to ensure that they all meet a minimum amount of general usefulness before being included. In this case, I would recommend adding it to the specific project that needs it and reuse it from there, but I'll defer to the existing package owners on the final decision.

**Comment by Blakesalot** [comment]
@MykhailoMolyboha, I should point out that Nova Charts does have a gauge component with radial (donut) and linear modes that was intended for use cases similar to this.

Here are the docs for that: https://nova-ui.solarwinds.io/#/charts/main/additional-documentation/chart-types/gauge.html
And, here is the source code: https://github.com/solarwinds/nova/tree/main/packages/charts/src/gauge

The look and feel are tailored to the Nova design spec, so it doesn’t **look** the same as the component in this PR, but it offers capabilities similar to those needed by your use case. And, it could potentially be enhanced to fulfill any requirements that it currently doesn’t meet.

**Review by findo11 — APPROVED** [review]

**Comment by findo11** [comment]
Approved, but please consider Blake's idea.

**Pull request closed without merging**
